concepticondata/conceptlists/Berry-1987-195.tsv

```diff
@@ -0,0 +1,196 @@
+ID	NUMBER	ENGLISH	LEXIBANK_GLOSS	INDONESIAN	CLASSIFICATION	PAGE	LOCATOR	CONCEPTICON_ID	CONCEPTICON_GLOSS
+Berry-1987-195-1	1	abdomen	abdomen	perut	NOUNS / Body Parts	62	44	1251	BELLY
+Berry-1987-195-2	2	back	back	punggung	NOUNS / Body Parts	62	44	1291	BACK
+Berry-1987-195-3	3	blood	blood	darah	NOUNS / Body Parts	62	44	946	BLOOD
+Berry-1987-195-4	4	body hair	body hair	bulu badan	NOUNS / Body Parts	62	44	189	HAIR (BODY)
+Berry-1987-195-5	5	bone	bone	tulang	NOUNS / Body Parts	63	45	1394	BONE
+Berry-1987-195-6	6	breast	breast	susu	NOUNS / Body Parts	63	45	1402	BREAST
+Berry-1987-195-7	7	ear	ear	telinga	NOUNS / Body Parts	63	45	1247	EAR
+Berry-1987-195-8	8	elbow	elbow	siku	NOUNS / Body Parts	63	45	981	ELBOW
+Berry-1987-195-9	9	eye	eye	mata	NOUNS / Body Parts	63	45	1248	EYE
+Berry-1987-195-10	10	finger	finger	jari	NOUNS / Body Parts	63	45	1303	FINGER
+Berry-1987-195-11	11	hair (head)	hair (head)	rambut	NOUNS / Body Parts	63	45	2648	HAIR (HEAD)
+Berry-1987-195-12	12	hand	hand	tangan	NOUNS / Body Parts	63	45	1277	HAND
+Berry-1987-195-13	13	head	head	kepala	NOUNS / Body Parts	63	45	1256	HEAD
+Berry-1987-195-14	14	heart	heart	jantung	NOUNS / Body Parts	63	45	1223	HEART
+Berry-1987-195-15	15	leg/foot	leg, foot	kaki	NOUNS / Body Parts	63	45	2098	FOOT OR LEG
+Berry-1987-195-16	16	meat/flesh	meat, flesh	daging	NOUNS / Body Parts	63	45	2615	FLESH OR MEAT
+Berry-1987-195-17	17	mouth	mouth	mulut	NOUNS / Body Parts	64	46	674	MOUTH
+Berry-1987-195-18	18	nail (finger)	nail (finger)	kuku	NOUNS / Body Parts	64	46	1258	FINGERNAIL
+Berry-1987-195-19	19	neck	neck	leher	NOUNS / Body Parts	64	46	1333	NECK
+Berry-1987-195-20	20	nose	nose	hidung	NOUNS / Body Parts	64	46	1221	NOSE
+Berry-1987-195-21	21	skin	skin	kulit	NOUNS / Body Parts	64	46	763	SKIN
+Berry-1987-195-22	22	teeth	teeth	gigi	NOUNS / Body Parts	64	46	1380	TOOTH
+Berry-1987-195-23	23	tongue	tongue	lidah	NOUNS / Body Parts	64	46	1205	TONGUE
+Berry-1987-195-24	24	bird	bird	burung	NOUNS / Animate Nouns	64	46	937	BIRD
+Berry-1987-195-25	25	cassowary	cassowary	kasuari	NOUNS / Animate Nouns	64	46	466	CASSOWARY
+Berry-1987-195-26	26	child	child	anak	NOUNS / Animate Nouns	64	46	2099	CHILD
+Berry-1987-195-27	27	dog	dog	anjing	NOUNS / Animate Nouns	64	46	2009	DOG
+Berry-1987-195-28	28	father	father	bapak	NOUNS / Animate Nouns	65	47	1217	FATHER
+Berry-1987-195-29	29	female	female	perempuan	NOUNS / Animate Nouns	65	47	2105	FEMALE PERSON
+Berry-1987-195-30	30	fish	fish	ikan	NOUNS / Animate Nouns	65	47	227	FISH
+Berry-1987-195-31	31	fly	fly	lalat	NOUNS / Animate Nouns	65	47	1504	FLY (INSECT)
+Berry-1987-195-32	32	husband	husband	suami	NOUNS / Animate Nouns	65	47	1200	HUSBAND
+Berry-1987-195-33	33	leech	leech	lintah	NOUNS / Animate Nouns	65	47	949	WATER LEECH
```

Comment:
I don't know if this is specific for these languages, but could this not be just [2273 LEECH](https://concepticon.clld.org/parameters/2273)?

Reply:
thank you, @MuffinLinwist!

Reply:
hi @LinguList again! whenever you have time, please let me know your thoughts on item 33 (leech - lintah), then we can merge :-)

Reply:
Right. I remembered that I myself added LEECH, since WATER LEECH was something like a synonym of LEECH for some time. Then I decided for WATER LEECH, as I thought LEECH was retired. But I better correct it :-)